app/steps/medicaid/step_navigation.rb

```diff
@@ -4,41 +4,42 @@ module Medicaid
-        Medicaid::AmountsOverviewController,
-        Medicaid::ConfirmationController,
-        Medicaid::ContactController,
-        Medicaid::ContactCurrentAddressController,
-        Medicaid::ContactEmailController,
-        Medicaid::ContactHomeAddressController,
-        Medicaid::ContactHomelessController,
-        Medicaid::ContactOtherAddressController,
-        Medicaid::ContactPhoneController,
-        Medicaid::ContactSocialSecurityController,
-        Medicaid::ContactSsDobController,
-        Medicaid::ContactTextMessagesController,
-        Medicaid::ExpensesAlimonyController,
-        Medicaid::ExpensesStudentLoanController,
+        Medicaid::IntroLocationController,
+        Medicaid::IntroLocationHelpController,
+        Medicaid::IntroNameController,
+        # multi-member Medicaid::IntroHouseholdController,
+        # multi-member Medicaid::IntroHouseholdMemberController,
+        Medicaid::IntroCollegeController,
+        Medicaid::IntroCitizenController,
+        # multi-member Medicaid::IntroCaretakerController,
+        # multi-member Medicaid::IntroCaretakerMemberController,
+        Medicaid::InsuranceCurrentController,
+        Medicaid::InsuranceCurrentTypeController,
+        Medicaid::InsuranceMedicalExpensesController,
         Medicaid::HealthDisabilityController,
         Medicaid::HealthPregnancyController,
+        Medicaid::TaxFilingController,
         Medicaid::IncomeJobController,
         Medicaid::IncomeJobNumberController,
+        Medicaid::IncomeSelfEmploymentController,
         Medicaid::IncomeOtherIncomeController,
         Medicaid::IncomeOtherIncomeTypeController,
-        Medicaid::IncomeSelfEmploymentController,
-        Medicaid::InsuranceCurrentController,
-        Medicaid::InsuranceCurrentTypeController,
-        Medicaid::InsuranceMedicalExpensesController,
-        Medicaid::IntroCaretakerController,
-        Medicaid::IntroCaretakerMemberController,
-        Medicaid::IntroCitizenController,
-        Medicaid::IntroHouseholdController,
-        Medicaid::IntroHouseholdMemberController,
-        Medicaid::IntroLocationController,
-        Medicaid::IntroLocationHelpController,
-        Medicaid::IntroNameController,
-        Medicaid::TaxFilingController,
+        Medicaid::ExpensesStudentLoanController,
+        Medicaid::ExpensesAlimonyController,
+        Medicaid::AmountsOverviewController,
+        Medicaid::AmountsIncomeController,
+        Medicaid::AmountsExpensesController,
+        Medicaid::ContactController,
+        Medicaid::ContactHomeAddressController,
+        Medicaid::ContactOtherAddressController,
+        Medicaid::ContactHomelessController,
+        Medicaid::ContactCurrentAddressController,
+        Medicaid::ContactPhoneController,
+        Medicaid::ContactTextMessagesController,
+        Medicaid::ContactEmailController,
+        Medicaid::ContactSsDobController,
+        Medicaid::ContactSocialSecurityController,
+        Medicaid::ConfirmationController,
```

Comment:
Do we know enough yet where we could divide these up into their own subsections? As in - not within the "Introduction" key?

Reply:
good idea! done at d7c7758